src/components/Cell/Cell.tsx

```diff
@@ -1,72 +1,105 @@
-    const handleCloseModal = () => {
-        setIsClicked(false)
-    }
+  const handlePlanted = () => {
+    setIsPlanted(true)
+  }
 
-    const handleClick = (e: React.MouseEvent) => {
-        const target = e.target as Element
-        if(!cellContents) {
-            setIsDisabled(!isDisabled)
-            !isDisabled ? target.classList.add('disabled') : target.classList.remove('disabled')
-        } else {
-            setIsClicked(true)
-            target.classList.add('disable-scale')
-            const parent = target.parentNode as Element
-            parent.classList.add('disable-hover')
-        }
-    }
+  const handleRemove = () => {
+    setIsPlanted(false)
+    setCellContents(undefined)
+  }
+
+  const handleCloseModal = () => {
+    setIsClicked(false)
+  }
 
-    const hoverStyle = isOver && !isDisabled && {
-        transform: 'scale(1.3)',
-        backgroundColor: 'LawnGreen'
+  const handleClick = (e: React.MouseEvent) => {
+    const target = e.target as Element
+    if (!cellContents) {
+      setIsDisabled(!isDisabled)
+      !isDisabled ? setClassName('cell disabled') : setClassName('cell');
+    } else {
+      setIsClicked(true)
+      target.classList.add('disable-scale')
+      const parent = target.parentNode as Element
+      parent.classList.add('disable-hover')
     }
+  }
+
+  const emptyCell = () : void => {
+    setIsDisabled(false);
+    setCellContents(undefined);
+    setIsPopulated(false);
+  }
+
+  const hoverStyle = isOver && !isDisabled && {
+    transform: 'scale(1.3)',
+    backgroundColor: 'LawnGreen'
+  }
 
-    const divStyle = !isDisabled && {
-        backgroundImage: `url(${cellContents?.plant.image})`,
-        backgroundPosition: 'center',
-        backgroundSize: '100%',
-        border: isPlanted ? 'solid LawnGreen 3px' : 'solid white 3px'
-    };
+  const divStyle = !isDisabled && {
+    backgroundImage: `url(${cellContents?.plant.image})`,
+    backgroundPosition: 'center',
+    backgroundSize: '100%',
+    border: isPlanted ? 'solid LawnGreen 3px' : 'solid white 3px'
+  };
 
-    return (
-        <div id={id} className='cell' style={{...divStyle, ...hoverStyle}} onClick={handleClick} ref={dropRef}>
-            {isClicked && <div className='cell-modal'>
-                {cellContents && <CellActions plant={cellContents?.plant} handlePlanted={handlePlanted} handleRemove={handleRemove} handleCloseModal={handleCloseModal}/>}
-            </div>}
+  return (
+    <>
+      {isPopulated ? (
+        <div id={id} className={className} style={{...divStyle, ...hoverStyle}} onClick={handleClick} ref={dropRef}>
+          {isClicked && <div className='cell-modal'>
+            {cellContents && <CellActions plant={cellContents?.plant} handlePlanted={handlePlanted} handleRemove={handleRemove} handleCloseModal={handleCloseModal}/>}
+          </div>}
         </div>
-    )
+      ) : (
+        <div id={id} className={className} style={{...hoverStyle}} onClick={handleClick} ref={dropRef}></div>
```

Comment:
i think we want to add `...divStyle` back to this condition as well.  This fixes the positioning issues, but the planted green border now remains even on an empty cell.